setup.py

```diff
@@ -10,7 +10,7 @@
 
 setuptools.setup(
     name="tf2_gnn",
-    version="2.2.2",
+    version="2.2.3",
```

Comment:
Sorry, forgot to say this earlier: This should bump the minor version, as you introduce a new feature.